src/08_comprehensions.py

```diff
@@ -2,22 +2,22 @@
 List comprehensions are one cool and unique feature of Python.
 They essentially act as a terse and concise way of initializing
 and populating a list given some expression that specifies how
-the list should be populated. 
+the list should be populated.
 
 Take a look at https://docs.python.org/3/tutorial/datastructures.html#list-comprehensions
 for more info regarding list comprehensions.
 """
 
 # Write a list comprehension to produce the array [1, 2, 3, 4, 5]
 
-y = []
+y = [number for number in range(1, 6)]
 
-print (y)
+print(y)
 
 # Write a list comprehension to produce the cubes of the numbers 0-9:
 # [0, 1, 8, 27, 64, 125, 216, 343, 512, 729]
 
-y = []
+y = [number ** 3 for number in range(0, 10)]
```

Comment:
It works perfectly but just a small thing to notice is that range() has 0 as it's default starting point. Like this:

y = [number ** 3 for number in range(10)]